internal/pkg/apps/install_test.go

```diff
@@ -55,7 +55,8 @@ func TestInstall(t *testing.T) {
 		mockBoltExperiment      bool
 		mockConfirmPrompt       bool
 		mockIsTTY               bool
-		mockManifest            types.SlackYaml
+		mockLocalManifest       types.SlackYaml
+		mockRemoteManifest      types.SlackYaml
```

Comment:
👁️‍🗨️ quibble: Some strange postfix notation might make this ordering more clear?

```suggestion
		mockManifestAppLocal    types.SlackYaml
		mockManifestAppRemote   types.SlackYaml
```

Reply:
Commit 013ad0d refactors the naming to `mockManifestAppLocal` and `mockManifestAppRemote` 🔨